apps/src/templates/sectionsRefresh/sections-refresh.module.scss

```diff
@@ -19,3 +21,24 @@
   display: flex;
   align-items: center;
 }
+
+.multiTables {
+  display: flex;
+}
+
+.table {
+  background-color: $neutral_dark10;
+  margin: 4px;
+  flex: 1 1 0px;
```

Comment:
This allows the column to have nearly the same width, while still optimizing for space if one has way more content than another.